test.cmake

```diff
@@ -142,7 +142,7 @@ macro(ADD_PYTHON_UNIT_TEST NAME SOURCE)
 
   set(MODULES "${ARGN}") # ARGN is not a variable
   foreach(MODULE_PATH IN LISTS MODULES)
-    list(APPEND PYTHONPATH "${CMAKE_BINARY_DIR}/${MODULE_PATH}")
+    cmake_path(APPEND PYTHONPATH "${CMAKE_BINARY_DIR}/${MODULE_PATH}")
```

Comment:
If I understand correctly the `cmake_path` documentation, because `${CMAKE_BINARY_DIR}/${MODULE_PATH}` is absolute, this is equivalent to:
```
set(PYTHONPATH "${CMAKE_BINARY_DIR}/${MODULE_PATH}")
```
which is entirely different from the original version so I don't get what this is fixing exactly?

Reply:
You are right. I am sorry, this solution was not the good one that I proposed.